pkgs/development/libraries/liburing/default.nix

```diff
@@ -4,25 +4,28 @@
 
 stdenv.mkDerivation rec {
   pname = "liburing";
-  version = "0.6pre600_${builtins.substring 0 8 src.rev}";
+  version = "0.7pre700_${builtins.substring 0 8 src.rev}";
```

Comment:
The number after the version appears to be the commit number, which in this case would be 654. On the other hand, I think it would be better to stick to the stable 0.6 release and just apply the ARM patch.